packages/cloud-cognitive/src/components/Datagrid/styles/addons/_CustomizeColumnsModal.scss

```diff
@@ -46,3 +61,22 @@
   position: relative; // needed for the react-dnd, otherwise the drag preview will not work correctly
   overflow: auto;
 }
+
+#customize-columns-select-all {
+  padding-left: $spacing-08;
+}
+
+.bx--checkbox-label {
```

Comment:
Use `$carbon-prefix` and also this rule should be scoped to the datagrid component, ie
```scss
.#{$block-class} .#{$carbon-prefix}--checkbox-label {
...
}
```

Reply:
replaced `.bx--checkbox-label` with `.#{$block-class} .#{$carbon-prefix}--checkbox-label`

Reply:
Also moved the rule to `datagrid.scss`